src/ConvertsToSeconds.php

```diff
@@ -0,0 +1,9 @@
+<?php
+declare(strict_types=1);

```

Comment:
Could we have this `declare` on the same line as the php tag?

Reply:
Sure